Source/common/CertificateHelpers.h

```diff
@@ -0,0 +1,47 @@
+/// Copyright 2023 Google LLC
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     https://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+#ifndef SANTA__COMMON__CERTIFICATEHELPERS_H
+#define SANTA__COMMON__CERTIFICATEHELPERS_H
```

Comment:
This shouldn't be needed - the impl is in a .m so this header can always be used with `#import`